### ISSUE TITLE
Setting up for dev/testing

### ISSUE THREAD

**Issue by jcasimir** [keavy/rubythere#1]
I have a few tweaks that I wanted to pitch in, but I'm having a difficult time getting the dev environment setup and tests running.  If you have time, would you consider checking out the code from Github, then make some notes about how to get it running?

I was particularly tripping up on Shoulda and FactoryGirl.  I think they're trying to use RSpec since it's loaded but then the test unit tests using "context" crash.

**Comment by keavy** [comment]
Yeah, sorry about that, have now revamped the old Shoulda based unit tests with Rspec ones. Currently have some issues with the Cucumber integration tests re auth, but...

Now a readme there with setup instructions. Let me know if I missed something

**Comment by jcasimir** [comment]
Awesome, I'll check it out next time I get some hack time.  It'll
likely be 2+ weeks :(
- JC

On Mon, Apr 25, 2011 at 6:16 AM, keavy
reply@reply.github.com
wrote:
